docs/api/permissions.rst

```diff
@@ -0,0 +1,332 @@
+| Configuration  | ``buckets:create``     | Ability to create new buckets.   |
+|                |                        |                                  |
++----------------+------------------------+----------------------------------+
+| Bucket         | ``write``              | Ability to write + read on the   |
+|                |                        | bucket and all children objects. |
+|                +------------------------+----------------------------------+
+|                | ``read``               | Ability to read all objects in   |
+|                |                        | the bucket.                      |
+|                +------------------------+----------------------------------+
+|                | ``collections:create`` | Ability to create new            |
+|                |                        | collections in the bucket.       |
+|                +------------------------+----------------------------------+
+|                | ``groups:create``      | Ability to create new groups     |
+|                |                        | in the bucket.                   |
++----------------+------------------------+----------------------------------+
+| Collection     | ``write``              | Ability to write and read all    |
+|                |                        | objects in the collection.       |
+|                +------------------------+----------------------------------+
+|                | ``read``               | Ability to read all objects in   |
+|                |                        | the collection.                  |
+|                +------------------------+----------------------------------+
+|                | ``records:create``     | Ability to create new records    |
+|                |                        | in the collection.               |
++----------------+------------------------+----------------------------------+
+| Record         | ``write``              | Ability to write and read the    |
+|                |                        | record.                          |
+|                +------------------------+----------------------------------+
+|                | ``read``               | Ability to read the record.      |
+|                |                        |                                  |
++----------------+------------------------+----------------------------------+
+| Group          | ``write``              | Ability to write and read the    |
+|                |                        | group.                           |
+|                +------------------------+----------------------------------+
+|                | ``read``               | Ability to read the group.       |
+|                |                        |                                  |
++----------------+------------------------+----------------------------------+
+
+
+By default the ``write`` permission is given to the creator of an
```

Comment:
I assume that the write permission is grated to the creator for the object that they created, though this is not stated explicitly. Consider something like:
> By default, the creator of an object is granted the write permission on that object.
> By default, newly created objects grant write permission to their creator.